pkg/scheduler/service/service.go

```diff
@@ -300,9 +302,11 @@ func (s *SchedulerServer) Run(_ []string) error {
 		log.Fatalf("Failed to start driver: %v", err)
 	}
 
-	//TODO(jdef) we need real task reconciliation at some point
-	log.V(1).Info("Clearing old pods from the registry")
-	clearOldPods(client)
+	/*
```

Comment:
Should we enable this code with a flag (e.g. `--clean_slate`)? If so, it would have to be invoked before mesosPodScheduler.Init() which is recovering tasks from the pod state.